src/bases/base.js

```diff
@@ -1,20 +1,8 @@
 import basex from '../../vendor/base-x.js'
 import { coerce } from '../bytes.js'
-
-/**
- * @typedef {import('./interface').BaseEncoder} BaseEncoder
- * @typedef {import('./interface').BaseDecoder} BaseDecoder
- * @typedef {import('./interface').BaseCodec} BaseCodec
- */
-
-/**
- * @template {string} T
- * @typedef {import('./interface').Multibase<T>} Multibase
- */
-/**
- * @template {string} T
- * @typedef {import('./interface').MultibaseEncoder<T>} MultibaseEncoder
- */
+// Linter can see that API is used in types.
```

Comment:
```suggestion
// Linter can't see that API is used in types.
```

you have 4 instances of this comment